docker/prod/Jenkinsfile

```diff
@@ -6,130 +6,150 @@ def loggerImage = "artifactory.wikia-inc.com/sus/mediawiki-logger"
 def rolloutStatus = 1
 
 def render(String template, java.util.LinkedHashMap binding) {
-    def engine = new groovy.text.SimpleTemplateEngine()
+	def engine = new groovy.text.SimpleTemplateEngine()
 
-    return engine.createTemplate(template).make(binding).toString()
+	return engine.createTemplate(template).make(binding).toString()
 }
 
 node("docker-daemon") {
-    def appBranch = params.app.split('\\.')[0]
-    def configBranch = params.config.split('\\.')[0]
-    def datacenter = params.datacenter
-    def environment = params.environment
+	def appBranch = params.app.split('\\.')[0]
+	def configBranch = params.config.split('\\.')[0]
+	def datacenter = params.datacenter
+	def environment = params.environment
 
-    def k8sContext = "kube-${datacenter}-${environment}"
+	def k8sContext = "kube-${datacenter}-${environment}"
 
-    def appHash
-    def configHash
-    def imageExists = false
-    def imageTag
+	def appHash
+	def configHash
+	def imageExists = false
+	def imageTag
 
-    def buildUser = "Anonymous"
+	def buildUser = "Anonymous"
 
-    stage("Clone repositories") {
+	stage("Prompt for a version") {
+		def deploymentInfo
+
+		withDockerContainer(kubectlImage) {
+			deploymentInfo = sh(
+				script: "kubectl --context kube-sjc-prod -n prod get deployments -l app=mediawiki-prod -o jsonpath='{.items[0].spec.template.metadata.labels}'",returnStdout: true
+			)
+		}
+
+		def prevConfigVersion = sh(script: 'echo "' + deploymentInfo + "\" | grep -o -P '(?<=config_version:)[^\\] ]*'", returnStdout: true)
+		def prevAppVersion = sh(script: 'echo "' + deploymentInfo + "\" | grep -o -P '(?<=app_version:)[^\\] ]*'", returnStdout: true)
+
+		timeout(time: 120, unit: 'SECONDS') {
+			userInput = input(
```

Comment:
What do we need it for? Is it meant as a chance to CTRL + C? In that case the name `"Prompt for a version"` is misleading.